themes/tiddlywiki/vanilla/base.tid

```diff
@@ -320,10 +324,40 @@ table tfoot tr td {
 	background-color: <<colour table-footer-background>>;
 }
 
+/*
+Table utility classes
+*/
+
+/* Remove borders from table as used in eg: GettingStarted*/
+.tc-table-no-border,
+.tc-table-no-border th,
+.tc-table-no-border td {
+	border: initial;
+}
```

Comment:
Saving some bytes:

.tc-table-no-border,
.tc-table-no-border :is(th, td) {
	border: initial;
}

Reply:
Yes indeed, we don't use `:is(x)` very often because it wasn't available in the IE days.